### PR TITLE
Client timeout adjustments

### PR DESCRIPTION
**Pull request by dtp263**
* make timeout configurable via env_var
* change timeout default from 10s to 30s

### CODE REVIEW

File: client/api.go [hunk 1]
```diff
@@ -71,7 +73,8 @@ type Client struct {
 }
 
 func NewClient(endpoint string) *Client {
-	timeout := time.Duration(10 * time.Second)
+	timeoutSeconds := viper.GetInt("CLIENT_TIMEOUT_SECONDS")
+	timeout := time.Duration(time.Duration(timeoutSeconds) * time.Second)
```
Comment (BisonAl) [inline client/api.go:77]:
```suggestion
	timeout := time.Duration(timeoutSeconds * time.Second)
```
Do we need that extra `time.Duration` cast? Looking at https://golang.org/pkg/time/#Duration

**Review by BisonAl — COMMENTED** [review]
had one note about time casting

File: client/api.go (hunk 1 above)
Comment (BisonAl) [reply client/api.go:77]: Apparently it doesn't make a difference output-wise if you check out https://play.golang.org/p/XTY3Y8aUxgG - but probs for the best to do less casting

Comment (dtp263) [reply client/api.go:77]: The issue there is you cant multiply a `time.Duration` by an int. its gotta be multiplied by another `time.Duration`


Screen Shot 2021-02-04 at 1 45 10 PM

**Comment by dtp263** [comment]
Tried to not cast. Heres the result.

Screen Shot 2021-02-04 at 2 07 59 PM

**Review by BisonAl — APPROVED** [review]

**Pull request merged**